v1/pipeline/stages/cd/environments-yaml.yaml

```diff
@@ -1,25 +1,27 @@
 title: EnvironmentsYaml
 type: object
 properties:
-  filters:
-    oneOf:
-    - type: array
-      items:
-        $ref: filter-yaml.yaml
-    - type: string
-      pattern: ^<\+input>((\.)((executionInput\(\))|(allowedValues|default|regex)\(.+?\)))*$
-      minLength: 1
-  metadata:
-    readOnly: true
-    $ref: environments-metadata.yaml
   values:
     oneOf:
-    - type: array
-      items:
-        $ref: environment-yaml-v2.yaml
-    - type: string
-      pattern: (<\+.+>.*)
-      minLength: 1
-  description:
-    desc: This is the description for EnvironmentsYaml
-$schema: http://json-schema.org/draft-07/schema#
+      - type: array
+        items:
+          $ref: environment-yaml.yaml
+      - type: string
+        pattern: ^<\+inputs\..*>$
+  inherit:
+    $ref: inherit-environment.yaml
+  filters:
+    type: array
+    items:
+      $ref: env-infra-filter-yaml.yaml
+  sequential:
+    oneOf:
```

Comment:
if string is allowed then why do we need oneOf boolean and string?
can't we just have type string?

Reply:
its because , in general practice use provide true or false with quotes and without quotes.
To support both of them, i have kept it both.

Its followed in v0, if we want to make more strict schema for booleans we can only have boolean, but i think both would be less error prone to user.

Reply:
i think we should make it strict to be boolean only, this oneOf would be okay if you want to allow expression

Reply:
My bad, didn't see the field.
We want to allow expression for "sequential" field.

Reply:
Discussed offline. resolving it.